crates/corro-pg/src/lib.rs

```diff
@@ -3378,598 +3352,3 @@ fn field_types(
 
     Ok(fields)
 }
-
-#[cfg(test)]
```

Comment:
I moved all the tests to an integration test file, this single file is massive, and would mean iterating on test code would rebuild the library as well, even if the change only occurred in test code.